spec/models/ext_management_system_spec.rb

```diff
@@ -258,6 +258,24 @@
     before(:each) do
       @ems = FactoryGirl.create(:ems_vmware)
       (1..2).each { |i| FactoryGirl.create(:vm_vmware, :ext_management_system => @ems, :name => "vm_#{i}") }
+      FactoryGirl.create(
+        :host_vmware,
+        :ext_management_system => @ems, :name => "host_1",
+        :hardware => FactoryGirl.create(:hardware, :cpu_total_cores => 4, :memory_mb => 1024)
+      )
```

Comment:
is it possible to move these statements into the actual `it` clauses below?

that way, you can see the setup and what happens for each of the tests.

The way it is now, it is setting up a bunch of fixture data and it is not clear what part of the fixture is necessary / needed.

think `:name` can be skipped.
aside: In another PR, we may also want to make an easier way to add `hardware` to a host/vm.